llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -5944,6 +5944,9 @@ SITargetLowering::EmitInstrWithCustomInserter(MachineInstr &MI,
     return lowerWaveReduce(MI, *BB, *getSubtarget(), AMDGPU::S_XOR_B64);
   case AMDGPU::S_UADDO_PSEUDO:
   case AMDGPU::S_USUBO_PSEUDO: {
+    MachineRegisterInfo &MRI = BB->getParent()->getRegInfo();
+    const GCNSubtarget &ST = MF->getSubtarget<GCNSubtarget>();
+    const SIRegisterInfo *TRI = ST.getRegisterInfo();
```

Comment:
```suggestion
    MachineRegisterInfo &MRI = BB->getParent()->getRegInfo();
    const SIRegisterInfo *TRI = Subtarget->getRegisterInfo();
```

Reply:
Change done in https://github.com/llvm/llvm-project/pull/159814